labellab-flask/requirements.txt.

```diff
@@ -0,0 +1,9 @@
+Click==7.0
+Flask==1.1.1
+Flask-Cors==3.0.8
+itsdangerous==1.1.0
+Jinja2==2.11.1
+MarkupSafe==1.1.1
+python-dotenv==0.11.0
+six==1.14.0
+Werkzeug==1.0.0
```

Comment:
Shall we add the packages as necessary?

Reply:
These were added when I had set-uped the project. I think we should keep them as they can come in handy as the development goes forward.